doc/sphinx-guides/source/installation/config.rst

```diff
@@ -408,7 +408,7 @@ By default Dataverse will attempt to connect to Solr on port 8983 on localhost.
 
 The relative path URL to which users will be sent after signup. The default setting is below.
 
-``curl -X PUT -d true /dataverseuser.xhtml?editMode=CREATE http://localhost:8080/api/admin/settings/:SignUpUrl``
```

Comment:
The "true" in this line shouldn't be there so I'm glad @donsizemore is proposing that it be removed.